jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -623,60 +623,7 @@ The ability to connect to multiple account integrations on CircleCI allows you t
 [#deprecated-system-environment-variables]
 == Deprecated system environment variables
 
-GitLab-based projects do not have the following system environment variables available. If your pipelines need these environment variables, we recommend you use suitable replacements from the available <<pipeline-variables#,pipeline values>>.
-
-[.table.table-striped]
-[cols=2*, options="header"]
-|===
-| Name
-| Description
-
-| `CI_PULL_REQUESTS`
-| Comma-separated list of URLs of the current build’s associated pull requests.
-
-| `CI_PULL_REQUEST`
-| The URL of the associated pull request. If there are multiple associated pull requests, one URL is randomly chosen.
-
-| `CIRCLE_PR_NUMBER`
-| The number of the associated GitHub or Bitbucket pull request. Only available on forked PRs.
-
-| `CIRCLE_PR_USERNAME`
-| The GitHub or Bitbucket username of the user who created the pull request. Only available on forked PRs.
-
-| `CIRCLE_PR_REPONAME`
-| The name of the GitHub or Bitbucket repository where the pull request was created. Only available on forked PRs.
-
-| `CIRCLE_PROJECT_USERNAME`
-| The GitHub or Bitbucket username of the current project.
-
-| `CIRCLE_PROJECT_REPONAME`
-| The name of the repository of the current project.
-
-| `CIRCLE_REPOSITORY_URL`
-| The URL of your GitHub or Bitbucket repository.
-
-| `CIRLCE_SHA1`
-| The SHA1 hash of the last commit of the current build.
-
-| `CIRCLE_TAG`
-| The name of the git tag, if the current build is tagged. For more information, see the xref:workflows#executing-workflows-for-a-git-tag[Git tag job execution] section of the Using workflows to orchestrate jobs page.
-
-|===
-
-If you must use these as environment variables in your pipelines, you can do so by using the xref:env-vars#environment-variable-usage-options[`environment` key] in your configuration and providing your own mappings:
-
-```yaml
-build:
-  docker:
-    - image: cimg/node:17.0
-      auth:
-        username: mydockerhub-user
-        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-  environment:
-    CIRCLE_PROJECT_REPONAME: << pipeline.trigger_parameters.gitlab.repo_name >>
-  steps:
-    - run: echo $CIRCLE_PROJECT_REPONAME
-```
+There are a number of xref:variables#built-in-environment-variables[built-in environment variables] that are not available in GitLab-based projects. If your pipelines need these environment variables, we recommend you use suitable replacements from the available xref:pipeline-variables#[pipeline values].
```

Comment:
Could we add something like "VCS support for each built-in environment variable is indicated in the table in the "Built-in environment variables" section" just to be really clear what we're linking people to? Just a suggestion